composer.json

```diff
@@ -34,12 +34,12 @@
         "phpspec/php-diff": "^1.1",
         "phpunit/phpunit": "^6.0",
         "polishsymfonycommunity/symfony-mocker-container": "^1.0",
-        "symfony/browser-kit": "^3.2",
-        "symfony/finder": "^3.2",
-        "symfony/framework-bundle": "^3.2"
+        "symfony/browser-kit": "^3.2|^4.0",
+        "symfony/finder": "^3.2|^4.0",
+        "symfony/framework-bundle": "^3.2|^4.0"
```

Comment:
We need to bump minimal Symfony version here as well, as `KERNEL_CLASS` has been added [in tag 3.3](https://github.com/symfony/symfony/blob/v3.3.0/src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php)